src/guides/hyperlinks.md

```diff
@@ -2,14 +2,16 @@
 A hyperlink is clickable-text. When the user clicks on the text, it directs them to a certain URL.
 
 ## General Hyperlinks
-You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md) and in [webhook](../guides/webhooks.md) content/description.
+You can use hyperlinks inside [`$description[]`](../bdscript/description.md), [`$addField[]`](../bdscript/addField.md), [webhook](../guides/webhooks.md) content/description, slash command response content and ephemeral message content.
```

Comment:
Add a link to the slash command guide.
Only slash command responses can be ephemeral, so `slash command response content` should be enough.

Reply:
An ephemeral can be called not only by the slash command, but also by interaction with a button, an option in a menu, and a modal. In general, just by an interaction (the slash command is also a kind of interaction, but without the `$onInteraction`).

Reply:
ephemeral.md should be updated. I think I will do this along with updating slashCommands.md. 
I have it planned.

Reply:
Oh

Reply:
Ephemeral messages aren't a different type of response, they are just a normal interaction response with a flag.
That's why I don't feel the need to mention them here.